src/test/kotlin/uk/gov/justice/digital/hmpps/supportadditionalneedsapi/integration/resource/UpdatePlanCreationScheduleStatusTest.kt

```diff
@@ -79,4 +80,63 @@ class UpdatePlanCreationScheduleStatusTest : IntegrationTestBase() {
     assertThat(actual!!.status).isEqualTo(HttpStatus.BAD_REQUEST.value())
     assertThat(actual.userMessage).isEqualTo("Reason must be specified for Exemptions")
   }
+
+  @Test
+  fun `update a plan creation schedule status to EXEMPT_PRISONER_NOT_COMPLY fail due to no schedule`() {
+    // Given
+    stubGetTokenFromHmppsAuth()
+    stubGetDisplayName("system")
+    val prisonNumber = randomValidPrisonNumber()
+    aValidPlanCreationScheduleExists(prisonNumber, status = COMPLETED)
+    val request = UpdatePlanCreationStatusRequest(
+      prisonId = "LWI",
+      exemptionReason = PlanCreationScheduleExemptionReason.EXEMPT_NOT_REQUIRED,
+      exemptionDetail = "because I say so",
+      status = PlanCreationUpdateStatus.EXEMPT_PRISONER_NOT_COMPLY,
+    )
+
+    // When
+    val response = webTestClient.patch()
+      .uri(URI_TEMPLATE, prisonNumber)
+      .headers(setAuthorisation(roles = listOf("ROLE_SUPPORT_ADDITIONAL_NEEDS__ELSP__RW"), username = "testuser"))
+      .bodyValue(request)
+      .exchange()
+      .expectStatus()
+      .is4xxClientError
+      .returnResult(ErrorResponse::class.java)
+
+    // Then
+    val actual = response.responseBody.blockFirst()
+    assertThat(actual!!.status).isEqualTo(HttpStatus.CONFLICT.value())
+    assertThat(actual.userMessage).isEqualTo("Plan creation schedule status must be [SCHEDULED] but was [COMPLETED] for prisoner [$prisonNumber]")
+  }
+
+  @Test
+  fun `update a plan creation schedule status to EXEMPT_PRISONER_NOT_COMPLY fail due to status not being SCHEDULED`() {
```

Comment:
Minor; the tests are right , but the test names are the wrong way round 🤣 
IE. this one's impl is about failing because the plan doesnt exist, but the test name is about the schedule being in the wrong status
And vica versa with the other test

🤣